eva_sub_cli/jinja_templates/fasta_check.html

```diff
@@ -35,22 +36,25 @@
         {% if 'metadata_assembly_compatible' in results_for_fasta %}
             <!-- found assembly in metadata, so definitely know the analysis -->
             {% set analysis_text = results_for_fasta.get('associated_analyses')|join(", ") %}
+            {% set expand_icon = "" %}
             {% if results_for_fasta.get('metadata_assembly_compatible') %}
                 {% set icon = "&#10004;" %}
                 {% set row_class = "report-section pass" %}
                 {% set text = analysis_text + ": Assembly accession in metadata is compatible" %}
             {% else %}
+                {% set expand_icon = "&#9654;" %}
                 {% set icon = "&#10060;" %}
                 {% set row_class = "report-section fail collapsible" %}
                 {% set text = analysis_text + ": Assembly accession in metadata is not compatible" %}
             {% endif %}
         {% else %}
             <!-- has possible assemblies but no metadata assembly -->
+            {% set expand_icon = "&#9654;" %}
             {% set icon = "&#10060;" %}
             {% set row_class = "report-section fail collapsible" %}
             {% set text = "No assembly accession found in metadata" %}
         {% endif %}
-        <div class='{{ row_class }}'>{{ icon }} {{ text }} </div>
+        <div class='{{ row_class }}'><span class="icon">{{ expand_icon }}</span> {{ icon }} {{ text }} </div>
```

Comment:
Could the class be called `expand-icon` so it's not confused with the other icon?

Reply:
done